src/main/java/woowa/sw/baseball/ResultCalcMachine.java

```diff
@@ -0,0 +1,33 @@
+package woowa.sw.baseball;
+
+import java.util.Map;
+
+/**
+ * 3자리 난수와 사용자가 입력한 숫자를 비교하는 class
+ * author : sw
+ */
+public class ResultCalcMachine {
+    private static final int[] BALL_IDX = {1, 2, 3};
+    private Map<Integer, Integer> randBalls;
+
+    public ResultCalcMachine(Map<Integer, Integer> randBalls) {
+        this.randBalls = randBalls;
+    }
+
+    public StrikeBallInfo resultCalc(Map<Integer, Integer> userBalls) {
+        StrikeBallInfo strikeBallInfo = new StrikeBallInfo(0, 0);
+        for (int numBallIdx : BALL_IDX) {
+            if (randBalls.get(numBallIdx).equals(userBalls.get(numBallIdx))) {
+                strikeBallInfo.setStrikeCnt(strikeBallInfo.getStrikeCnt()+1);
```

Comment:
```suggestion
                strikeBallInfo.setStrikeCnt(strikeBallInfo.getStrikeCnt() + 1);
```
<br>
자바 컨벤션에서 연산자 앞, 뒤는 한칸씩 띄워줘야 합니다.
이클립스나 인텔리제이 모두 자동 포맷팅 기능이 있는데, 써보시면 좋을 것 같아요.